win32ss/gdi/eng/pdevobj.c

```diff
@@ -548,6 +600,15 @@ PDEVOBJ_Create(
         return NULL;
     }
 
+    /* Enable DirectDraw */
+    if (!PDEVOBJ_bEnableDirectDraw(ppdev))
+    {
+        DPRINT1("Failed to enable DirectDraw\n");
```

Comment:
DPRINTs or ERR? (the debug channels are available in this file)

Reply:
Okay, `ERR` seems better here. :slightly_smiling_face: